src/pudl/transform/ferc1.py

```diff
@@ -311,6 +362,60 @@ def transform_end(self, df: pd.DataFrame) -> pd.DataFrame:
         """Enforce the database schema and remove any cached dataframes."""
         return self.enforce_schema(df)
 
+    def normalize_metadata_xbrl(
+        self, xbrl_fact_names: list[str] | None
```

Comment:
i know this is xbrl's naming convention but in our context "facts" don't really have meaning. column names?